cmake/packaging/rpm.cmake

```diff
@@ -0,0 +1,306 @@
+
+    set(pseudo_plugins_recommends "${pseudo_plugins_recommends}" PARENT_SCOPE)
+    set(all_plugins_suggest "${all_plugins_suggest}" PARENT_SCOPE)
+    set(installed_plugins "${installed_plugins}" PARENT_SCOPE)
+endfunction()
+
+macro(ov_cpack_settings)
+    # fill a list of components which are part of rpm
+    set(cpack_components_all ${CPACK_COMPONENTS_ALL})
+    unset(CPACK_COMPONENTS_ALL)
+    foreach(item IN LISTS cpack_components_all)
+        # filter out some components, which are not needed to be wrapped to .deb package
+        if(# NOT ${item} MATCHES ".*(python).*" AND
+           # python wheels are not needed to be wrapped by rpm packages
+           NOT item STREQUAL OV_CPACK_COMP_PYTHON_WHEELS AND
+           # even for case of system TBB we have installation rules for wheels packages
+           # so, need to skip this explicitly
+           NOT item MATCHES "^tbb(_dev)?$" AND
+           # the same for pugixml
+           NOT item STREQUAL "pugixml" AND
+           # we have copyright file for rpm package
+           NOT item STREQUAL OV_CPACK_COMP_LICENSING AND
+           # not appropriate components
+           NOT item STREQUAL OV_CPACK_COMP_DEPLOYMENT_MANAGER AND
+           NOT item STREQUAL OV_CPACK_COMP_INSTALL_DEPENDENCIES AND
+           NOT item STREQUAL OV_CPACK_COMP_SETUPVARS)
+            list(APPEND CPACK_COMPONENTS_ALL ${item})
+        endif()
+    endforeach()
+    list(REMOVE_DUPLICATES CPACK_COMPONENTS_ALL)
+
+    # version with 3 components
+    set(cpack_name_ver "${OpenVINO_VERSION}")
+    # full version with epoch and release components
+    set(cpack_full_ver "${CPACK_PACKAGE_VERSION}")
+
+    # take release version into account
+    if(DEFINED CPACK_RPM_PACKAGE_RELEASE)
+        set(cpack_full_ver "${cpack_full_ver}-${CPACK_RPM_PACKAGE_RELEASE}")
+    endif()
+
+    # take epoch version into account
+    if(DEFINED CPACK_RPM_PACKAGE_EPOCH)
+        set(cpack_full_ver "${CPACK_RPM_PACKAGE_EPOCH}:${cpack_full_ver}")
+    endif()
+
+    # a list of conflicting package versions
+    set(conflicting_versions
+        # 2022 release series
+        # - 2022.1.0 is the last public release with rpm packages from Intel install team
+        # - 2022.1.1 does not have rpm packages enabled, distributed only as archives
+        2022.1.0)
+
+    # core
+    set(CPACK_COMPONENT_CORE_DESCRIPTION "OpenVINO C / C++ Runtime libraries")
+    set(CPACK_RPM_CORE_PACKAGE_NAME "libopenvino-${cpack_name_ver}")
+    # we need triggers to run ldconfig for openvino
+    set(CPACK_RPM_CORE_PACKAGE_CONTROL_EXTRA "${def_postinst};${def_postrm};${def_triggers}")
+    # use lintian to check packages in post-build step
+    set(CPACK_POST_BUILD_SCRIPTS "${IEDevScripts_DIR}/packaging/rpm_post_build.cmake")
+
+    # We currently don't have versioning for openvino core library
+    ov_rpm_add_rpmlint_suppression(core
+        "shlib-without-versioned-soname"
+        "package-name-doesnt-match-sonames")
```

Comment:
does it have the same error messages as lintian? maybe we need to remove it? or suppress different tags